utils/instructions/Dual/index.ts

```diff
@@ -88,13 +88,13 @@ export async function getConfigInstruction({
   schema,
   setFormErrors,
 }: StakingOptionArgs): Promise<UiInstruction> {
-  const isValid = await validateInstruction({ schema, form, setFormErrors })
+  // const isValid = await validateInstruction({ schema, form, setFormErrors })
```

Comment:
we should keep validation. modify the schema how you want it updated

Reply:
https://github.com/Dual-Finance/governance-ui/blob/9e5327c43719d24d1adc1ee63bb958f450330936/utils/validations.tsx#L777

Here?

Reply:
thats the gso one. might want this
https://github.com/Dual-Finance/governance-ui/blob/9e5327c43719d24d1adc1ee63bb958f450330936/utils/validations.tsx#L728

but if you think this is a necessary change for both, do them both

Reply:
Does it fail there or on 721?

Reply:
changed to string entry